client/homepage/layout.js

```diff
@@ -133,6 +125,10 @@ Layout.propTypes = {
 	 * If the task list option is being fetched.
 	 */
 	requestingTaskList: PropTypes.bool.isRequired,
+	/**
+	 * If the task list has been completed.
+	 */
+	taskListComplete: PropTypes.bool,
```

Comment:
Theres a proptype violation here, "Invalid prop `taskListComplete` of type `string` supplied to `Layout`, expected `boolean`." I found it when the task list wasn't visible.

Reply:
I'm unable to reproduce this - can you provide steps?